pkg/azuredisk/nodeserver.go

```diff
@@ -261,10 +261,9 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "failed to find device path with LUN %s. %v", lun, err)
 		}
-		klog.V(2).Infof("NodePublishVolume [block]: found device path %s with LUN %s", source, lun)
-		err = d.ensureBlockTargetFile(target)
-		if err != nil {
-			return nil, err
+		klog.V(2).Infof("NodePublishVolume [block]: found device path %s with lun %s", source, lun)
```

Comment:
Same change for nodeserver_v2?

Reply:
In `nodeserver_v2.go`, `lun` is an `int32`. I could change this to `%d` but `%v` is equivalent.

Reply:
Oh, I see. "LUN" got changed to a lowercase "lun". The `nodeserver.go` change seems wrong to me since LUN is an acronym. Currently, we seem to be very inconsistent with its capilization.

Reply:
Also, return the error code internal here? https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/0226b3d8ddd3347975a1b4eb32cc20218f459eb3/pkg/azuredisk/nodeserver_v2.go#L333

Reply:
This is a call to the NodeProvisioner in v2 which already returns gRPC statuses. The lowercase name is the v1 implementation.